modules/net-vpc/subnets.tf

```diff
@@ -130,11 +132,16 @@ resource "google_compute_subnetwork" "proxy_only" {
   ip_cidr_range = each.value.ip_cidr_range
   description = (
     each.value.description == null
-    ? "Terraform-managed proxy-only subnet for Regional HTTPS or Internal HTTPS LB."
+    ? "Terraform-managed proxy-only subnet for Regional HTTPS, Internal HTTPS or Cross-Regional HTTPS Internal LB."
     : each.value.description
   )
-  purpose = "REGIONAL_MANAGED_PROXY"
-  role    = each.value.active != false ? "ACTIVE" : "BACKUP"
+  purpose = try(
+    each.value.purpose,
+    each.value.global != false
+    ? "GLOBAL_MANAGED_PROXY"
+  : "REGIONAL_MANAGED_PROXY")
```

Comment:
nit: can you add a line break between `"REGIONAL_MANAGED_PROXY"` and the following `)` then run `terraform ftm`?

Reply:
done!